components/server/src/server.rs

```diff
@@ -1658,23 +1644,39 @@ where
         let range_cache_engine_context =
             RangeCacheEngineContext::new(range_cache_engine_config.clone(), self.pd_client.clone());
         let range_cache_engine_statistics = range_cache_engine_context.statistics();
-        let kv_engine: EK = KvEngineBuilder::build(
-            range_cache_engine_context,
-            disk_engine.clone(),
-            Some(self.pd_client.clone()),
-            Some(Arc::new(self.region_info_accessor.clone())),
-        );
+        let in_memory_engine = if self.core.config.range_cache_engine.enabled {
+            let in_memory_engine = build_hybrid_engine(
```

Comment:
Do you plan to remove hybrid enigne or rename it in the following PR? I think now this name is confusing.

Reply:
Yes, will do clean up in https://github.com/tikv/tikv/pull/17380.